scalafix/input/src/main/scala/fix/v1_0_0/RemoveSplit.scala

```diff
@@ -0,0 +1,17 @@
+/*
+rewrite = "scala:fix.v1_0_0.RemoveSplit"
+ */
+package fix
+package to1_0_0
+
+import cats.implicits._
+import cats.arrow.Split
+import cats.syntax.split._
```

Comment:
ah, good catch, thanks. That's a half-baked attempt that I accidentally left in there.
I just pushed a proper version of that fix.

By the way, it'd be nice if scalafix tests were run by the CI (so that these kind of mistake would be caught). Do you think it's possible to add them to the build?